mslice/tests/workspace_provider_test.py

```diff
@@ -1,90 +1,93 @@
 from __future__ import (absolute_import, division, print_function)
 import numpy as np
+from mock import patch
 import unittest
-from mantid.simpleapi import CreateWorkspace, CreateSimulationWorkspace, AddSampleLog, ConvertToMD
-from mslice.models.workspacemanager.mantid_workspace_provider import MantidWorkspaceProvider
+from mantid.simpleapi import CreateSimulationWorkspace, AddSampleLog, ConvertToMD, CloneWorkspace
+from mslice.models.workspacemanager.mantid_workspace_provider import (wrap_workspace, delete_workspace,
+                                                                      get_workspace_names, subtract,
+                                                                      get_workspace_handle, add_workspace_runs,
+                                                                      combine_workspace, rename_workspace,
+                                                                      propagate_properties, get_limits)
+from mslice.models.workspacemanager.mantid_workspace_provider import _processEfixed
 
 class MantidWorkspaceProviderTest(unittest.TestCase):
 
     def setUp(self):
-        self.ws_provider = MantidWorkspaceProvider()
-
-        self.test_ws_2d = CreateSimulationWorkspace(Instrument='MAR', BinParams=[-10, 1, 10],
-                                                    UnitX='DeltaE', OutputWorkspace='test_ws_2d')
-        AddSampleLog(self.test_ws_2d, LogName='Ei', LogText='3.', LogType='Number')
-        self.test_ws_md = ConvertToMD(InputWorkspace=self.test_ws_2d, OutputWorkspace="test_ws_md", QDimensions='|Q|',
+        self.test_ws_2d = wrap_workspace(CreateSimulationWorkspace(Instrument='MAR', BinParams=[-10, 1, 10],
```

Comment:
I guess it could get a little repetitive to keep having to call `wrap_workspace` every time we call an algorithm. Can we centralise this somehow? Maybe a single function to execute a mantid algorithm by string name and this could also centralise the `StoreInTheADS` call?